docs/source/features/sheet_management/dataframe.rst

```diff
@@ -4,3 +4,125 @@
 ###########
  DataFrame
 ###########
+
+:class:`~graviti.dataframe.frame.DataFrame` is an integrated data structure
+with an easy-to-use API for simplifying data processing in Dataset.
+A Graviti DataFrame contains the columns Protex schema defines and the data resides in the rows and columns.
+
+************************
+ Initialize a DataFrame
+************************
+
+Initialize a Dataframe by a List:
```

Comment:
do we need to add the restrictions about the list? like the descriptions about DataFrame in pandas  https://pandas.pydata.org/docs/user_guide/dsintro.html#dataframe